magento/pwa-studio
# Added handle add to cart feature in category page

## Description

Added feature to add an item to the cart from the category page. The action is only available on Simple products. For configurable products, the button simply takes them to the PDP page, and for all other products the button is disabled.

## Related Issue

Closes PWA-1843.

### Verification Stakeholders

@revanth0212 

### Verification Steps

1. Open the deployed app.
2. Go to a simple product in the category page. Click on the add to cart button and verify the product is added to the cart.
3. Go to a configurable product. Click on the add to cart button and verify that the user is taken to the PDP page.
4. Go to any other product and verify that the button is disabled.

## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
